source/community/release_notes/palm.rst

```diff
@@ -15,30 +15,237 @@ These are the release notes for the Palm release, the 16th community release of
+
+
+
+Other Studio Updates
+====================
+
+* Using the setting, :code:`FEATURES['ENABLE_CREATOR_GROUP']`, users with CourseCreate permission can choose their organization from a drop-down in Studio. 
+* Drag and Drop has been moved out of advanced problem types, and now appears along side other problems in Studio.  
+* The "jump_to_id" hint in the Unit View will now pre-fill with the current location ID. This should avoid course authors copying and pasting the generic text. 
+
 
 Administrators & Operators
 **************************
 
+Experimental New Learner Home Page 
+==================================
+
+The new Learner Home has many of the same features of the old learner dashboard, with some extended functionality and 
+performance enhancements.
+
+* The Learner Home is now built with Paragon, the Open edX design pattern library. It is accessible and easy to style with brand colors. 
+* Course cards show the course thumbnail, information about the course, and the ability to upgrade to a paid track or view/begin a course (if applicable). Further course actions (e.g. unenroll, email opt-out settings, and social media share) have been moved to the menu/triple dot icon on the course card.
+* Clicking the “Refine” button opens options to filter by course status or sort either by most recent enrollment (default) or title.
+
+.. image:: /_images/community/release_notes/palm/new_learner_home_filtering.png
+    :alt: The Refine pop-up with Course State and Sort options
+
+* All of a user’s courses are fetched on page load. To make the page manageable, we paginate that list, showing 25 courses at a time. To view other courses, a user should page through their list of courses using the pagination controls at the bottom of the course list.
+
+.. image:: /_images/community/release_notes/palm/new_learner_home_pagination.png
+    :alt: Next, previous and page number buttons appear below the list of courses
+
+* Site staff can now masquerade as users on the platform by typing a username or email in the “View as” box and hitting submit. This is designed to be “view only” so change actions (e.g. enroll, unenroll, selecting a session) are blocked in this view.
+* There is a complete list of changes in the `Open edX wiki <https://openedx.atlassian.net/wiki/spaces/OEPM/blog/2022/11/21/3584589831/2U+New+Learner+Home+Page#Comparison-with-old-experience>`_. 
+
+To turn on the New Learner home page, add and enable the :code:`learner_home_mfe.enabled` Waffle Flag in the django 
+admin. To support a phased role out, there is also a setting :code:`LEARNER_HOME_MFE_REDIRECT_PERCENTAGE`. Set it to an 
+integer value (0 - 100) representing what percentage of users should be redirected to Learner Home. 0 represents no 
+rollout, 100 represents rollout to all users.
```

Comment:
I suggest rewording this to point to the (hot-off-the-oven) experimental Tutor plugin installation instructions:

https://github.com/openedx/openedx-tutor-plugins/tree/main/plugins/tutor-contrib-learner-dashboard-mfe#learner-dashboard-mfe-plugin-for-tutor

Reply:
Thanks. I replaced the instructions with your suggested link, and moved it up to the start of the section.